pkg/kubectl/describe.go

```diff
@@ -1766,23 +1766,42 @@ func (d *ServiceAccountDescriber) Describe(namespace, name string, describerSett
 
 	tokens := []api.Secret{}
 
-	tokenSelector := fields.SelectorFromSet(map[string]string{api.SecretTypeField: string(api.SecretTypeServiceAccountToken)})
-	options := api.ListOptions{FieldSelector: tokenSelector}
-	secrets, err := d.Core().Secrets(namespace).List(options)
+	// remainingSecrets is the set of all secrets remaining on a
+	// service account that are not present in the "tokens" slice.
+	remainingSecrets := sets.NewString()
+	missingSecrets := sets.NewString()
+
+	secrets, err := d.Core().Secrets(namespace).List(api.ListOptions{})
 	if err == nil {
```

Comment:
comment about why we tolerate an error (need to be able to describe a service account even if referenced secrets can't be fetched)